server/src/models/user.js

```diff
@@ -0,0 +1,88 @@
+import mongoose from "mongoose";
+import validator from "validator";
+import bcrypt from "bcryptjs";
+import jwt from "jsonwebtoken";
+
+const userSchema = mongoose.Schema({
+  fName: {
```

Comment:
it will be more declarative if you use first_name or firstName instead of fName.